composer.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "paypro/magento2",
   "description": "PayPro Payment Module for Magento 2",
-  "version": "1.0.3",
+  "version": "1.0.4",
```

Comment:
Here we define it as `2.0.0`: https://github.com/paypronl/magento2-payments-plugin/blob/master/etc/module.xml#L3. I think we should have them identical.